feature-libs/organization/schematics/add-organization/index_spec.ts

```diff
@@ -104,249 +106,173 @@ describe('Spartacus Organization schematics: ng-add', () => {
       beforeEach(async () => {
         appTree = await schematicRunner
-          .runSchematicAsync('ng-add', defaultFeatureOptions, appTree)
+          .runSchematicAsync(
+            'ng-add',
+            { ...defaultFeatureOptions, lazy: false },
+            appTree
+          )
           .toPromise();
       });
 
-      it('should import the i18n resource and chunk from assets', async () => {
-        const administrationModule = appTree.readContent(
-          administrationFeatureModulePath
-        );
-        expect(administrationModule).toContain(
-          `import { organizationTranslationChunksConfig, organizationTranslations } from "@spartacus/organization/administration/assets";`
-        );
-      });
-      it('should provideConfig', async () => {
-        const administrationModule = appTree.readContent(
-          administrationFeatureModulePath
-        );
-        expect(administrationModule).toContain(
-          `resources: organizationTranslations,`
-        );
-        expect(administrationModule).toContain(
-          `chunks: organizationTranslationChunksConfig,`
-        );
+      it('should import appropriate modules', async () => {
+        const module = appTree.readContent(administrationFeatureModulePath);
+        expect(module).toMatchSnapshot();
       });
     });
   });
 
   describe('Order approval feature', () => {
-    describe('styling', () => {
-      beforeEach(async () => {
-        appTree = await schematicRunner
-          .runSchematicAsync('ng-add', defaultFeatureOptions, appTree)
-          .toPromise();
-      });
-
-      it('should create a proper scss file', () => {
-        const scssContent = appTree.readContent(scssFilePath);
-        expect(scssContent).toMatchSnapshot();
-      });
-
-      it('should update angular.json', async () => {
-        const content = appTree.readContent('/angular.json');
-        expect(content).toMatchSnapshot();
-      });
-    });
-
-    describe('eager loading', () => {
+    describe('general setup', () => {
       beforeEach(async () => {
         appTree = await schematicRunner
           .runSchematicAsync(
             'ng-add',
-            { ...defaultFeatureOptions, lazy: false },
+            {
+              ...defaultFeatureOptions,
+              features: [CLI_ORDER_APPROVAL_FEATURE],
+            },
             appTree
           )
           .toPromise();
       });
 
-      it('should import appropriate modules', async () => {
-        const orderApprovalModule = appTree.readContent(
-          orderApprovalFeatureModulePath
-        );
-        expect(orderApprovalModule).toContain(
-          `import { OrderApprovalRootModule } from "@spartacus/organization/order-approval/root";`
-        );
-        expect(orderApprovalModule).toContain(
-          `import { OrderApprovalModule } from "@spartacus/organization/order-approval";`
-        );
+      it('should install necessary Spartacus libraries', () => {
```

Comment:
Can we have this test not tied to any particular feature? So it's run only once in case of libraries with multiple features as it's not really tied to any feature schematics config.